### PR TITLE
Use string.IndexOf(char) instead of string.IndexOf(string) where possible

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomSerializerBase.cs [hunk 1]
```diff
@@ -80,7 +80,7 @@ private static string GetTypeNameFromCodeTypeReferenceHelper(IDesignerSerializat
 
             //create the MyGeneric`2[ part
             StringBuilder typename = new StringBuilder(typeref.BaseType);
-            if (!typeref.BaseType.Contains("`"))
+            if (typeref.BaseType.IndexOf('`') == -1)
```
Comment (danmosemsft) [inline src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomSerializerBase.cs]: This code will only ever run on .NET Core, and 3.0+ at that. So you can safely use `string.Contains(char)` without risk of it compiling to Linq as it does on .NET Framework.

Comment (danmosemsft) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomSerializerBase.cs]: https://apisof.net/catalog/System.String.Contains(Char)

Comment (hughbe) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomSerializerBase.cs]: Fixed!

**Comment by danmosemsft** [comment]
thanks!

**Pull request merged**